### ISSUE TITLE
[1.1.3] Install script error when executed from another directory

### ISSUE THREAD

**Issue by newtork** [deajan/osync#73]
- GNU bash, version 4.3.30(1)-release (x86_64-pc-linux-gnu)
- Debian Version 8.6

Downloaded and extracted from
```https://github.com/newtork/group-domain/archive/v1.1.3.tar.gz```

Error, when outside directory:
```
root@debian:~# ./osync-1.1.3/install.sh
Created directory [/etc/osync].
cp: cannot stat â./osync.shâ: No such file or directory
Cannot copy osync.sh to [/usr/local/bin]. Make sure to run install script in the directory containing all other files.
Also make sure you have permissions to write to [/usr/local/bin].
```

No problem, when inside directory:
```
root@debian:~/osync-1.1.3# ./install.sh
[...]
osync installed. Use with /usr/local/bin/osync
[...]
```

**Comment by deajan** [comment]
Well that's why there is a " Make sure to run install script in the directory containing all other files." message yet.
I'll have this handled for next release.

**Comment by newtork** [comment]
You are right, it's more of an inconvenience than anything.
Keep it the good work, really great project! :)

**Comment by deajan** [comment]
Solved for v1.2, can't be bothered to backport to v1.1 :)
